notebooks/en/trl_grpo_reasoning_advanced_reward.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        processing_class=tokenizer,
</pre><p>you can remove this line, GRPO will load automatically the tokenizer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>